mlx-sys/build.rs

```diff
@@ -8,6 +8,7 @@ fn main() {
     let mut config = Config::new("src/mlx-c");
     config.very_verbose(true);
     config.define("CMAKE_INSTALL_PREFIX", ".");
+    config.define("CMAKE_BUILD_TYPE", "Release");
```

Comment:
Should check for debug_assertions to check whether we're in release or debug, no?

Reply:
Yeah, probably should

Reply:
Resolved in [39071b9](https://github.com/oxideai/mlx-rs/pull/87/commits/39071b9b0a6cf1a054492c98f8bbac9dd27cccbf)